components/brave_wallet/browser/internal/hd_key.cc

```diff
@@ -430,6 +431,35 @@ std::vector<uint8_t> HDKey::GetPublicKeyFromX25519_XSalsa20_Poly1305() const {
   return public_key;
 }
 
+std::vector<uint8_t> HDKey::DecryptCipherFromX25519_XSalsa20_Poly1305(
```

Comment:
Ahh interesting they only use the decrypt function as well instead of [decrypt_safely](https://github.com/MetaMask/eth-sig-util/blob/73ace3309bf4b97d901fb66cd61db15eede7afe9/src/encryption.ts#L230) which they authored but didn't use.